src/main/java/edu/kis/vh/nursery/IStackInterface.java

```diff
@@ -0,0 +1,13 @@
+package edu.kis.vh.nursery;
+
+public interface IStackInterface {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.